autopep8.bat

```diff
@@ -0,0 +1,3 @@
+python -m autopep8 nafas --recursive --aggressive --aggressive --in-place --pep8-passes 2000 --verbose
+python -m autopep8 setup.py --recursive --aggressive --aggressive --in-place --pep8-passes 2000 --verbose
+python -m autopep8 otherfiles --recursive --aggressive --aggressive --in-place --pep8-passes 2000 --verbose
```

Comment:
Could you pleas make `.sh` version of this file too, anyway I think I can add it to your `build_script` branch too.
I don't know if it's necessary or not but there is a newline miss here.

Reply:
Solved in https://github.com/sepandhaghighi/nafas/pull/10/commits/1c8c9c154b68c2094e5e9327b11ad7aa263ff325